citrination_client/client.py

```diff
@@ -304,6 +304,35 @@ def upload_file(self, file_path, dataset_id, root_path=None):
         """
         return self.upload(dataset_id, file_path)
 
+    def _data_analysis(self, model_name):
```

Comment:
I don't think the raw output from the dev-api is user friendly, which is why I "hid" it.  In a future PR, we can expose the correlations as another method.